### DISCUSSION TITLE
Covariate on Clearence, do measurement units matter?

### DISCUSSION THREAD

**Discussion by dan-adi** [metrumresearchgroup/mrgsolve#1029] [Q&A]
I was looking at the following example code: [https://github.com/mrgsolve/examples/blob/master/PrTS/models/popmodel.cpp](url)

On line 17:
```
double LTVCL = THETA1 + THETA6 *log(BMI/25) + THETA8 *SEX + THETA7*log(EGFR/100);
```
EGFR is a covariate on Clearence, but CL is usually given as units of  L/h, and I assume the 100 value is the mean EGFR value measured in mL/min/1.73m^2. Shouldn't we keep units the same? I suppose EGFR/6?
I did try in a simpler 1 cmt model this:
```
double adjustedCL = exp(log(CL)  + log(EGFR/100));
```
vs this:
```
double adjustedCL = exp(log(CL)  + log(EGFR/6));
```
And, as expected, I was getting different results, so I assume units do matter.

**Comment by Mutaz94** [comment]
This is a scaling issue. The first adjustedCL has a population CL of L/hr/100efgr unit while the second one is L/hr/6efgr unit. So check your population values.
